colin/core/ruleset/loader.py

```diff
@@ -27,10 +27,10 @@
 
 def get_ruleset_struct_from_fileobj(fileobj):
     try:
-        logger.debug("Loading ruleset from file '{}'.".format(fileobj.name))
+        logger.debug(f"Loading ruleset from file '{fileobj.name}'.")
```

Comment:
W1202: Use % formatting in logging functions and pass the % parameters as arguments (logging-format-interpolation)

Reply:
Ah, missed that one..;(

Reply:
Fixed.